### ISSUE TITLE
GTCE tools couldnt work

### ISSUE THREAD

**Issue by xly2000** [codetaylor/dropt#57]
oredictionary and tag too

**Comment by codetaylor** [comment]
It is difficult for me to understand anything about this issue because it is described using so few words.

Please elaborate.

**Comment by xly2000** [comment]
sry. i mean tool restriction.

```
Dropt.list("grave")
    .priority(0)
    .add(Dropt.rule()
        .matchDrops([<minecraft:gravel>])
        .matchHarvester(Dropt.harvester()
              .type("PLAYER")
              .mainHand("WHITELIST", [<gregtech:meta_tool:16>.withTag({"GT.ToolStats": {PrimaryMaterial: "darmstadtium"}})], "shovel;0;-1")
      )
	    .addDrop(Dropt.drop()
            .selector(Dropt.weight(50))
            .items([<minecraft:gravel>])	  
         )
        .addDrop(Dropt.drop()
		    .selector(Dropt.weight(35))
            .items([<pyrotech:rock>])
        )
	    .addDrop(Dropt.drop()
            .selector(Dropt.weight(15))
            .items([<minecraft:flint>])  
		)	
);
```

normal shovels is  work, but gtce tool not

**Comment by codetaylor** [comment]
Ah, the NBT matching.

If I remember right, Dropt doesn't support NBT matching. I'll look into this in the future though.

**Comment by codetaylor** [comment]
The held item NBT matching feature will be available in `1.16.0`
